presto/dbapi.py

```diff
@@ -298,13 +300,45 @@ def _format_prepared_param(self, param):
         Formats parameters to be passed in an
         EXECUTE statement.
         """
+        if param is None:
+            return "NULL"
+        if isinstance(param, bool):
+            return "true" if param else "false"
+        if isinstance(param, int):
+            # TODO represent numbers exceeding 64-bit (BIGINT) as DECIMAL
+            return "%d" % param
+        if isinstance(param, float) and param == float("+inf"):
+            return "infinity()"
+        if isinstance(param, float) and param == float("-inf"):
+            return "-infinity()"
+        if isinstance(param, float) and math.isnan(param):
+            return "nan()"
+        if isinstance(param, float):
+            return "DOUBLE '%s'" % param
         if isinstance(param, str):
             return ("'%s'" % param.replace("'", "''"))
-        else:
-            # TODO: param could have many other types which should
-            #       be covered including unicode, maps, lists, etc.
-            #       https://github.com/prestosql/presto-python-client/issues/45
-            return str(param)
+        if isinstance(param, bytes):
+            return "X'%s'" % param.hex()
+        if isinstance(param, datetime.datetime):
+            datetime_str = param.strftime("%Y-%m-%d %H:%M:%S.%f %Z").rstrip(" ")
```

Comment:
the `rstrip` is now on the next line

```suggestion
            datetime_str = param.strftime("%Y-%m-%d %H:%M:%S.%f %Z")
```